js/ui/save-html.js

```diff
@@ -31,15 +31,19 @@ define(
                     ,   textAlign:      "center"
                     ,   fontSize:       "inherit"
                     }
-                ,   addButton = function (title, content, fileName, popupContent) {
+                ,   addButton = function (title, href, fileName, popupContent, type) {
+                        if(!type){
+                            href = "data:text/html;charset=utf-8," + encodeURIComponent(href)
+                       }
                         if (supportsDownload) {
                             $("<a></a>")
                                 .appendTo($div)
                                 .text(title)
                                 .css(buttonCSS)
                                 .attr({
-                                    href:   "data:text/html;charset=utf-8," + encodeURIComponent(content)
-                                ,   download:   fileName
+                                    href: href
+                                ,   download: fileName
+                                ,   type: type
```

Comment:
Well, it shows up in the diff, so must have been added by someone. :) I suppose this comes from @marcoscaceres.